aten/src/ATen/function_wrapper.py

```diff
@@ -553,6 +594,7 @@ def __getitem__(self, x):
     'type_method_formals': List[str],
     'variants': str,
     'with_gil': bool,
+    'when_sparse_dispatch': str,
```

Comment:
what's this

Reply:
this looks like a botched merge, I removed this recently.